### ISSUE TITLE
Duplicate gedit keyboard dhortcuts open GTK+ inspector.

### ISSUE THREAD

**Issue by SMed79** [hannenz/duplicate#4]
Hi, 
on **gedit 3.16.2** i have change plugin loader to **python3** and now it enabled but i have problem with keyboard shortcut ! 
[Ctrl]+[D] : Delete the current line.
[Ctrl]+[Shift]+[D] : open GTK+ inspector.

![gedit 3.16.2](https://cloud.githubusercontent.com/assets/1873139/7894845/5837083e-066a-11e5-80f1-f5851d4bf51e.png)

![GTK+ inspector](https://cloud.githubusercontent.com/assets/1873139/7894847/5bbb2ba2-066a-11e5-8510-136513b57acb.png)

Thanks!

**Comment by rugk** [comment]
Yeah,. I've got the same problem. See https://unix.stackexchange.com/questions/320682/how-to-get-a-duplicate-line-hot-key-in-gedit

**Comment by SMed79** [comment]
@rugk Post/request ignored ... i have an alternative for you ... **medit** http://mooedit.sourceforge.net/ 

Tools > add 'Script Lua'

```
    orign = doc.get_line_text()
    endposition = doc.get_pos_at_line_end(doc.get_line_at_cursor())
    doc.insert_text('\n' .. orign, endposition)
```
source: http://www.manjaro.fr/forum/viewtopic.php?p=62157#p62157

![medit](http://img110.xooimage.com/files/0/0/8/capture-d-cran---...17-35-34-493cdb1.png)
